src/newsletter.py

```diff
@@ -0,0 +1,30 @@
+import os
+from collections import namedtuple
+
+import httpx
+
+buttondown_api_key = os.getenv("BUTTONDOWN_API_KEY")
+header = {"Authorization": f"Token {buttondown_api_key}"}
```

Comment:
These feel like they belong in the function body or as function arguments rather than at the top level of the module. As is, this feels like it might be a pain to test and such if they're defined at time of import rather than time of use.

Reply:
in testing they are monkey patched normally, but I agree they could move into the function body